extra/1-currency-conversion.js

```diff
@@ -15,7 +17,9 @@ function convertToUSD() {}
   They have also decided that they should add a 1% fee to all foreign transactions, which means you only convert 99% of the £ to BRL.
 */
 
-function convertToBRL() {}
+function convertToBRL(amountGBP) {
+  return Number((0.99 * amountGBP * 5.7).toFixed(2));
```

Comment:
Suggestion: Avoid using the toFixed for math operations, as toFixed converts the number to a string. For a pure math function it is better to use the `Math.round(val)` function to avoid the conversion to a string.